CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
Okay, I had misunderstood earlier what you meant. Think I have added to the CHANGELOG the missing part.